MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -707,6 +732,42 @@ - (void)showDistributeDisabledAlert {
   });
 }
 
+- (void)showUpdateSetupFailedAlert:(NSString *)errorMessage {
```

Comment:
Just one very picky comment, there is a convention across the SDKs where empty lines within code blocs should always be followed by a comment. For instance:
```
  int a = 1;

  // Awesome example.
  a++; 
```

Reply:
ahh I see, thanks for the info! I like that convention 😄